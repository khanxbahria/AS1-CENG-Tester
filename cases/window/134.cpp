#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->print();
Tab L = Tab("vvv", "jj","ii");
wind->moveActiveTabTo(5);
Tab S = Tab("x", "lll","mmm");
wind->print();
wind->newTab(L);
Tab W = Tab("zzz", "kkk","a");
wind->isEmpty();
Tab P = Tab("pp", "cc","rr");
wind->getActiveTab();
Tab O = Tab("kkk", "cc","oo");
wind->getActiveTab();
wind->newTab(O);
Tab X = Tab("f", "nn","i");
wind->changeActiveTabTo(1);
Tab Y = Tab("w", "n","h");
wind->print();
Tab G = Tab("fff", "vv","ooo");
wind->getActiveTab();
Tab C = Tab("i", "hhh","rr");
wind->isEmpty();
wind->closeTab();
wind->changeActiveTabTo(3);
wind->changeActiveTabTo(2);
wind->getActiveTab();
Tab J = Tab("lll", "mmm","bbb");
wind->print();
Tab E = Tab("hh", "ss","ss");
wind->getActiveTab();
wind->getActiveTab();
Tab M = Tab("jjj", "mmm","b");
wind->getActiveTab();
Tab T = Tab("w", "ggg","rrr");
wind->isEmpty();
Tab D = Tab("z", "w","z");
wind->closeTab();
Tab V = Tab("bbb", "xxx","www");
wind->changeActiveTabTo(1);
wind->closeTab();
wind->isEmpty();
wind->isEmpty();
Tab F = Tab("lll", "fff","fff");
wind->closeTab();
wind->isEmpty();
wind->print();
Tab Q = Tab("w", "mm","ooo");
wind->print();
wind->print();
wind->print();
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(1);
wind->isEmpty();
wind->getActiveTab();
wind->newTab(C);
wind->print();
wind->print();
wind->moveActiveTabTo(5);
wind->print();
Tab N = Tab("rr", "f","bb");
wind->getActiveTab();
Tab H = Tab("zz", "qqq","aa");
wind->isEmpty();
Tab U = Tab("fff", "qq","ww");
wind->changeActiveTabTo(1);
wind->isEmpty();
wind->newTab(D);
wind->closeTab();
Tab K = Tab("sss", "uuu","xx");
wind->moveActiveTabTo(1);
Tab Z = Tab("ii", "iii","i");
wind->isEmpty();
wind->newTab(E);
wind->getActiveTab();
wind->moveActiveTabTo(3);
wind->isEmpty();
wind->newTab(F);
wind->changeActiveTabTo(0);
wind->print();
wind->closeTab();
wind->newTab(G);
wind->isEmpty();
wind->newTab(H);
wind->isEmpty();
wind->closeTab();
Tab A = Tab("v", "hhh","t");
wind->moveActiveTabTo(5);
wind->getActiveTab();
Tab B = Tab("mm", "uuu","ggg");
wind->print();
wind->changeActiveTabTo(2);
wind->changeActiveTabTo(3);
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
wind->changeActiveTabTo(4);
wind->newTab(A);
wind->moveActiveTabTo(2);
Tab I = Tab("p", "qqq","eee");
wind->isEmpty();
wind->newTab(B);
wind->print();
wind->getActiveTab();
wind->closeTab();
wind->moveActiveTabTo(2);
wind->closeTab();
wind->closeTab();
wind->newTab(I);
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(4);
wind->isEmpty();
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(2);
wind->changeActiveTabTo(0);
wind->moveActiveTabTo(5);
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->moveActiveTabTo(4);
wind->changeActiveTabTo(1);
wind->isEmpty();
wind->print();
wind->closeTab();
wind->newTab(J);
wind->closeTab();
wind->closeTab();
wind->newTab(K);
delete wind;
}
	return 0;
}