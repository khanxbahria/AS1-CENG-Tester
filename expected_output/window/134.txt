The active tab is tab 0




The active tab is tab 0




The active tab is tab 1




jj
vvv
ii

cc
kkk
oo

The active tab is tab 1




cc
kkk
oo

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
hhh
i
rr

The active tab is tab 0
hhh
i
rr

The active tab is tab 0
hhh
i
rr

The active tab is tab 0
hhh
i
rr

ss
hh
ss

fff
lll
fff

The active tab is tab 2
ss
hh
ss

vv
fff
ooo

fff
lll
fff

The active tab is tab 3
ss
hh
ss

vv
fff
ooo

hhh
v
t

uuu
mm
ggg

The active tab is tab 1
qqq
p
eee

ss
hh
ss

